topics/teaching/tutorials/workshop-final-exercise/tutorial.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: tutorial_hands_on
+
+title: "Train-the-Trainer: putting it all together"
+subtopic: pedagogy
+time_estimation: 60m
+questions:
+- 
+objectives:
+- 
+key_points:
+- 
+requirements:
+-
```

Comment:
test is failing currently because these are empty. Anything you can fill in here?

Reply:
I need to work on this tutorial. it is still WIP